app/views/coronavirus_landing_page/components/shared/_accordion_sections.html.erb

```diff
@@ -53,19 +51,21 @@
   border_top: border_top,
   font_size: "m"
 } if heading %>
-<div data-module="gem-track-click gtm-click-tracking">
+<div data-module="gtm-click-tracking">
```

Comment:
This wrapping div isn't needed since you're now passing the GTM tracking module directly to the component.